grails-app/domain/com/unifina/domain/Canvas.groovy

```diff
@@ -70,7 +70,7 @@ class Canvas {
 		adhoc defaultValue: false
 		runner index: 'runner_idx'
 		dashboardItems cascade: 'all-delete-orphan'
-		exampleType enumType: "identity", defaultValue: ExampleType.NOT_SET, index: 'example_type_idx'
+		exampleType enumType: "ordinal", index: "example_type_idx"
```

Comment:
Why we don't define the defaultValue here? (and 3 other places in this PR)

Reply:
@teogeb It doesn't do anything with enums

Reply:
How about if we set defaultValue: `ExampleType.NOT_SET.ordinal()`?